contributors/devel/e2e-tests.md

```diff
@@ -98,7 +103,8 @@ go run hack/e2e.go -- -v --up
 go run hack/e2e.go -- -v --test
 
 # Run tests matching the regex "\[Feature:Performance\]"
-go run hack/e2e.go -- -v --test --test_args="--ginkgo.focus=\[Feature:Performance\]"
+# Specify "--provider=local" flag when running the tests locally
```

Comment:
can you merge this into the above comment? eg:
`Run tests matching the regex ... against a local cluster`